From 8900bacb116cf6e653d9ddbc8338ad42da2006a8 Mon Sep 17 00:00:00 2001
From: Abhisek Devkota <ciwrl@lineageos.org>
Date: Sat, 3 Feb 2018 20:28:50 -0800
Subject: [PATCH 4/9] SUW: Don't export our WizardManager

Even though we disable this class when GMS is present, the
 intent-filters are still active and cause an activity resolver
 to be presented to the user. Disable export of this activity to
 work around this.

Change-Id: I5888c71b5fdb95e59e15366b66f56e7968e51f1a
---
 AndroidManifest.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 587140f..574f652 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -68,7 +68,7 @@
         <activity android:theme="@style/NoDisplay"
                   android:label="@string/activity_label_empty"
                   android:name=".wizardmanager.WizardManager"
-                  android:exported="true"
+                  android:exported="false"
                   android:excludeFromRecents="true"
                   android:configChanges="keyboard|keyboardHidden|mcc|mnc|orientation|screenSize"
                   android:immersive="true">
-- 
2.7.4

